training/src/test/scala/com/airbnb/aerosolve/training/DecisionTreeModelTest.scala

```diff
@@ -44,6 +44,16 @@ class DecisionTreeModelTest {
   }
 
   @Test
+  def testDecisionTreeTrainerMulticlassHellinger() = {
+    testDecisionTreeMulticlassTrainer("multiclass_hellinger", 0.5)
```

Comment:
The randomness makes the accuracy jump up and down and could make the test flaky. Since random is 0.25 I figured twice of random should be a good cutoff